tests/integration_tests_control_files/taylor_green_vortex_integration/viscous_TGV_energy_check_strong_adaptive_time_step.prm

```diff
@@ -13,6 +13,9 @@ set use_weak_form = false
 # Note: this was added to turn off check_same_coords() -- has no other function when dim!=1
 set use_periodic_bc = true
 
+# memory allocation
```

Comment:
Maybe a more descriptive comment? Imagine explaining this flag to a new grad student.

Reply:
fixed